meta/src/bin/main.rs

```diff
@@ -98,8 +100,14 @@ pub async fn start_service(opt: Opt) -> std::io::Result<()> {
             .service(api::restore)
             .service(api::debug)
```

Comment:
Mask the operating system inside the function

Reply:
Resolved.